src/reducers/versions.tsx

```diff
@@ -194,16 +205,26 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
   switch (action.type) {
     case getType(actions.loadVersionInfo): {
       const { version } = action.payload;
+
       return {
         ...state,
         versionInfo: {
           ...state.versionInfo,
           [version.id]: createInternalVersion(version),
         },
+        versionFiles: {
```

Comment:
I wasn't really sure that we needed to do this, which is one of the reasons I didn't do it initially, because when we load a version with the default file we probably won't be displaying it. But I guess there's nothing really wrong with doing this, and it does give us the option to display it if we want to.

Reply:
I think we could display it as it is the default file.